modules/proc_deploy-quay-openshift-operator.adoc

```diff
@@ -1120,32 +1300,215 @@ run the operator locally:
 $ operator-sdk up local --namespace=quay-enterprise
 ```
 
-= Upgrading {productname} and Clair
+= Upgrading {productname} 
+The Quay Operator v3.3.0 has many changes from v1.0.2. The most notable which
+affects the upgrade process is the backwards-incompatible change to the CRD.
+Ultimately, the CR (Custom Resource) used to deploy {productname} using the operator
+may have to be modified accordingly.
+
+== Upgrade Prerequisites
+Ensure that your deployment is using a supported persistance layer and
+database. A production {productname} deployment run by the Operator should *not* be
+relying on the Postgres instance or a OpenShift volume that has
+been created by the Operator.
+
+If you are using a Postgres instance or OpenShift volume that was created
+by the Operator, the upgrade path is not suported as the removal of the old
+Operator will cascade the deletion of your database and volume. It may be
+possible to manually migrate your data to supported storage mechanisms but
+this is not within the scope of the typical, or supported, upgrade path.
+
+Please read through the entire guide before following any steps as this upgrade
+path is potentially destructive and there is no guaranteed roll-back mechanism.
+
+== Upgrade Process Summary
+
+Here are the basic steps for upgrading the {productname} cluster
+you originally deployed from the v1.0.2 Quay Setup Operator to
+the v3.3 Quay Operator:
+
+. Document all configuration related to your current deployment.
+. Copy your CR and modify any configuration values as needed.
+. Remove your current deployment using `oc delete -f deployment.yaml`
```

Comment:
What does `deployment.yaml` contain? @kurtismullins could you help clarify, thanks

Reply:
Good question. I don't know.

Reply:
My understanding is that's the deployment.yaml file generated during the backup, now it was used to remove the deployment.

Reply:
Ah, so this assumes that you created a "deployment.yaml" at some point, but this procedure does not show that file being created. Kurtis is on vacation at the moment. I'll have to check with him to find out what he meant. I'll leave this open for the moment.